addOns/reports/src/main/java/org/zaproxy/addon/reports/ReportHelper.java

```diff
@@ -27,13 +27,42 @@
 import org.parosproxy.paros.Constant;
 import org.parosproxy.paros.control.Control;
 import org.parosproxy.paros.core.scanner.Alert;
+import org.zaproxy.addon.reports.sarif.SarifReportDataSupport;
 import org.zaproxy.zap.extension.alert.AlertNode;
 import org.zaproxy.zap.extension.stats.ExtensionStats;
 import org.zaproxy.zap.extension.stats.InMemoryStats;
 import org.zaproxy.zap.utils.XMLStringUtil;
 
 public class ReportHelper {
 
+    /**
+     * For some reports there is additional tooling necessary - e.g. for SARIF: Because the data
+     * structure is completely different to standard OWASP Zap model and would lead to unreadable
```

Comment:
Its always "ZAP" never "Zap" ;)

Reply:
Okay, will make a case sensitive search for "Zap" occurrences in the PR and change it accordingly.

Reply:
Ups... found extreme many parts in complete code, even in existing parts like `Constants.getZapHome()` etc. So will concentrate only on parts mentioned inside review instead of former mentioned "search and replace" . I think ZAP will only be used when it's inside a documentation or a comment - but usage in code (e.g. method names) this rule does not apply. Am I correct?

Reply:
Correct